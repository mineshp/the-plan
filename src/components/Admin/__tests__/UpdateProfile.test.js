import React from 'react';
import renderer from 'react-test-renderer';
import UpdateProfileComponent from '../UpdateProfile';

const mockHandleSubmit = jest.fn();
const mockHandleProfileNameChange = jest.fn();
const mockHandleProfileActiveChange = jest.fn();

describe('Admin > Manage > CreateProfiles', () => {
    it('renders Create Profile component', () => {
        const tree = renderer.create(
            <UpdateProfileComponent
                handleSubmit={mockHandleSubmit}
                handleProfileNameChange={mockHandleProfileNameChange}
                handleProfileActiveChange={mockHandleProfileActiveChange}
            />
        );
        expect(tree).toMatchSnapshot();
    });
});
