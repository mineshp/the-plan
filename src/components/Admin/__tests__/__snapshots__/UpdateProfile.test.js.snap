// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Admin > Manage > CreateProfiles renders Create Profile component 1`] = `
<div
  className="ui container"
>
  <form
    action={undefined}
    className="ui form"
    onSubmit={[Function]}
  >
    <div
      className="field"
    >
      <div
        className="ui input text-box-single-col-min"
      >
        <input
          defaultValue=""
          disabled={undefined}
          onChange={[Function]}
          placeholder="Profile Name..."
          tabIndex={undefined}
          type="text"
        />
      </div>
    </div>
    <div
      className="field"
    >
      <div
        className="ui checkbox"
        onChange={[Function]}
        onClick={[Function]}
        onMouseDown={[Function]}
        required={undefined}
      >
        <input
          checked={false}
          className="hidden"
          name="activeProfile"
          readOnly={true}
          tabIndex={0}
          type="checkbox"
          value={undefined}
        />
        <label>
          Profile Active
        </label>
      </div>
    </div>
    <button
      className="ui green button"
      disabled={undefined}
      onClick={[Function]}
      role="button"
      tabIndex={undefined}
      type="submit"
    >
      Save
    </button>
  </form>
</div>
`;
